C4G.Unity/Assets/C4G/Tests/Editor/CodeGeneratorTests.cs

```diff
@@ -0,0 +1,219 @@
+using System;
+using System.Collections.Generic;
+using C4G.Editor;
+using NUnit.Framework;
+
+namespace C4G.Tests.Editor
+{
+    public class CodeGeneratorTests
+    {
+        [Test]
+        public void GenerateDTOClass_UsualCase()
+        {
+            // Arrange
+            string expectedOutput =
+@"public partial class ClassName
+{
+    public int Id { get; set; }
+    public int BaseHp { get; set; }
+    public float BaseDamage { get; set; }
+    public string Name { get; set; }
+    public double DamagePerLevel { get; set; }
+}
+";
+            var className = "ClassName";
+            var propertyInfos = new List<ParsedPropertyInfo>
+            {
+                new ParsedPropertyInfo("Id", "int"),
+                new ParsedPropertyInfo("BaseHp", "int"),
+                new ParsedPropertyInfo("BaseDamage", "float"),
+                new ParsedPropertyInfo("Name", "string"),
+                new ParsedPropertyInfo("DamagePerLevel", "double"),
```

Comment:
To test the case you don't need all 6 lines and properties to setup.
In this way you have to write and read more lines of code to understand what you really testing now.

I suggest to make all tests with minimal setup to test without any additional "nice to have" elements.

Like:
If we test class name
```csharp
public partial class SomeClassName
{
// no need in extra properties
}
```
If we test order:
```csharp
public partial class SomeClassName
{
public string ShouldBeSecond {get;set}
public int SouldBeFirst {get; set}
}

What do you think?)

Reply:
Yep, i agree
Simplified setup code